neo/rawio/blackrockrawio.py

```diff
@@ -297,12 +297,33 @@ def _parse_header(self):
             # for nsxdef get_analogsignal_shape(self, block_index, seg_index):
             self.__nsx_data_header[nsx_nb] = self.__nsx_dataheader_reader[spec](nsx_nb)
 
-        # We can load only one for one class instance
-        if self.nsx_to_load is None and len(self._avail_nsx) > 0:
-            self.nsx_to_load = max(self._avail_nsx)
+        # nsx_to_load can be either int, list, 'max', all' (aka None)
+        # here make a list only
+        if self.nsx_to_load is None or self.nsx_to_load == 'all':
+            self.nsx_to_load = list(self._avail_nsx)
+        elif self.nsx_to_load == 'max':
+            if len(self._avail_nsx):
+                self.nsx_to_load = [max(self._avail_nsx)]
+            else:
+                self.nsx_to_load = []
+        elif isinstance(self.nsx_to_load, int):
+            self.nsx_to_load = [self.nsx_to_load]
+        elif isinstance(self.nsx_to_load, list):
+            pass
+        else:
+            raise(ValueError('nsx_to_load is wrong'))
+
+        assert all(nsx_nb in self._avail_nsx for nsx_nb in self.nsx_to_load),\
+                                    'nsx_to_load to not match available nsx list'
```

Comment:
I think there is a typo here.
```suggestion
                                    'nsx_to_load do not match available nsx list'
```